spec/lib/workos/configuration_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+# typed: false
+
+describe WorkOS do
+  describe '.configure' do
+    context 'with key and no timeout' do
+      before do
+        WorkOS.configure do |config|
+          config.key = 'example_api_key'
+        end
+      end
+      it 'sets the key and default timeout configuration' do
```

Comment:
Tiny nitpick: Can we insert a blank line between test setup and the test themselves?

```suggestion
      end
      
      it 'sets the key and default timeout configuration' do
```